docs/ref/contrib/admin/index.txt

```diff
@@ -1939,6 +1988,30 @@ return the uncompressed versions of the various JavaScript files, including
 
 .. _jQuery: https://jquery.com
 
+.. _contrib-admin-select2:
+
+jQuery Select2
```

Comment:
My intentions where to document that the namespace `window.django.jQuery.select2` is taken. But I can also leave this undocumented. I don't thing that anyone will run into conflicts because of that.